### PR TITLE
Validation: Fixed image-alt-text-duplicated issues

### PR DESCRIPTION
**Pull request by JustPies**
Fixed:

- image-alt-text-duplicated errors (duplicate alt text descriptions within an article)
- code-block-indented errors

This PR fixes 75 issues across 1 files and is part of ongoing Validation cleanup efforts. No other content has been edited or added.

### CODE REVIEW

File: dynamics-gp/payroll/PayrollUS.md [hunk 1]
```diff
@@ -1868,19 +1868,19 @@ In the **Employee Deduction Maintenance** window:
 | Federal | FEDCS |
 | State | AKCSWHLTH |
 
-![A screenshot again](media/DEDCHILDSUP1.JPG)
+![Screenshot of the window, showing deduction entries for example child, named CHILD.](media/DEDCHILDSUP1.JPG)
```
Comment (MeeraDi) [inline dynamics-gp/payroll/PayrollUS.md:1871]: for an example child, named CHILD.

File: dynamics-gp/payroll/PayrollUS.md [hunk 2]
```diff
@@ -1934,19 +1934,19 @@ In the **Employee Deduction Maintenance** window:
 | Federal | FEDCEDIT |
 | State | STATECRED |
 
-![A screenshot again](media/CREDITOR.JPG)
+![Screenshot of the Employee Deduction Maintenance window, showing default entries and empty input boxes.](media/CREDITOR.JPG)
 
 In the **Garnishment Maximum Setup** window:
 
-![one screenshot ](media/CREDITOR1.JPG)
+![Screenshot of a table above two Garnishment Maximum Setup windows. The table lists codes, method, and wages information. One window is for Federal credit, and the other is for state credit.](media/CREDITOR1.JPG)
```
Comment (MeeraDi) [inline dynamics-gp/payroll/PayrollUS.md:1941]: The table lists code, method, and wage information.

File: dynamics-gp/payroll/PayrollUS.md [hunk 3]
```diff
@@ -1934,19 +1934,19 @@ In the **Employee Deduction Maintenance** window:
...
 
 In the  **Earnings Setup** window:
 
-![Another screenshot ](media/CREDITOR2.JPG)
+![Screenshot of a table above two Earnings Setup windows. The table lists codes, deductions, and taxes information. One window is for Federal credit, and the other is for state credit.](media/CREDITOR2.JPG)
```
Comment (MeeraDi) [inline dynamics-gp/payroll/PayrollUS.md:1945]: The table lists code, deduction, and tax information.

File: dynamics-gp/payroll/PayrollUS.md [hunk 4]
```diff
@@ -2442,7 +2442,7 @@ the tax rate won't be applied when the maximum amount is reached.
 1. Open the Payroll Local Tax Setup window. (HR & Payroll \>\> Setup \>\>
     Payroll \>\> Local Tax)
 
-![A screenshot ](media/18fcb2fe462d9b4e41beb175bc0256f4.jpg)
+![Screenshot of window, showing Check Maximum Witholding entered into the Sequence box.](media/18fcb2fe462d9b4e41beb175bc0256f4.jpg)
```
Comment (MeeraDi) [inline dynamics-gp/payroll/PayrollUS.md:2445]: Screenshot of the payroll local tax setup window

File: dynamics-gp/payroll/PayrollUS.md [hunk 5]
```diff
@@ -1956,7 +1956,7 @@ Federal Creditor Garnishment Withholding Maximum:  Use the lesser of:
 •	(The amount by which earnings are greater than 30 times the Federal Minimum Wage) (for bi-weekly 60 times the Federal Minimum Wage) 
 60 x $7.25= $435.00		$480.77-$435.00 = $45.77
 
-![Also a screenshot ](media/CREDITOR4.JPG)
+![Screenshot of a window above a report, showing a net wage of three hundred and three dollars and three cents after deductions and taxes.](media/CREDITOR4.JPG)
```
Comment (MeeraDi) [inline dynamics-gp/payroll/PayrollUS.md:1959]: Screenshot of the earnings setup window

File: dynamics-gp/payroll/PayrollUS.md [hunk 6]
```diff
@@ -2911,7 +2911,7 @@ tracking tips* for more information.
 1. Open the Employee Maintenance window. (HR & Payroll \>\> Cards \>\> Payroll
     \>\> Employee)
 
-    ![A screenshot](media/6a0e4daae9eb17c33aa058f859855385.jpg)
+    ![Screenshot of the window, showing the information of an example employee named Pilar Ackerman.](media/6a0e4daae9eb17c33aa058f859855385.jpg)
```
Comment (MeeraDi) [inline dynamics-gp/payroll/PayrollUS.md:2914]: Screenshot of the employee maintenance window

File: dynamics-gp/payroll/PayrollUS.md [hunk 7]
```diff
@@ -3016,7 +3016,7 @@ hours per year, and posting account information.
 2. Enter or select an employee ID and choose Additional Information to open the
     Employee Additional Information Maintenance window.
 
-![A screenshot ](media/0a325f909f1f1ee2d85f5d4a4be08034.jpg)
+![Screenshot of the window, showing example employee Ackerman's birthdate, minimum net pay, and general personal information.](media/0a325f909f1f1ee2d85f5d4a4be08034.jpg)
```
Comment (MeeraDi) [inline dynamics-gp/payroll/PayrollUS.md:3019]: Screenshot of the employee additional information maintenance window

File: dynamics-gp/payroll/PayrollUS.md [hunk 8]
```diff
@@ -1868,19 +1868,19 @@ In the **Employee Deduction Maintenance** window:
 | Federal | FEDCS |
 | State | AKCSWHLTH |
 
-![A screenshot again](media/DEDCHILDSUP1.JPG)
+![Screenshot of the window, showing deduction entries for an example child, named CHILD.](media/DEDCHILDSUP1.JPG)
```
Comment (MeeraDi) [inline dynamics-gp/payroll/PayrollUS.md:1871]: Add name of window please

**Comment by JustPies** [comment]
**Global effort to fix validation errors**

 @edupont04   

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Review by edupont04 — APPROVED** [review]

**Pull request merged**